completions/bash/kpod

```diff
@@ -335,33 +354,75 @@ _kpod_save() {
      --format
      "
      local boolean_options="
-     --quiet -q
+     --quiet
+     -q
+     --help
+     -h
      "
      _complete_ "$options_with_args" "$boolean_options"
 }
 
 _kpod_export() {
      local options_with_args="
-     --output -o
+     --output
+     -o
      "
      local boolean_options="
+     --help
+     -h
      "
      _complete_ "$options_with_args" "$boolean_options"
 }
 
 _kpod_ps() {
      local options_with_args="
-     --filter -f
+     --filter
+     -f
      --format
-     --last -n
+     --last
+     -n
      "
      local boolean_options="
-     --all -a
-     --latest -l
+     --all
```

Comment:
Split out this cleanup stuff into a separate pull request.

Reply:
fixed